test/dotnet-publish.Tests/PublishAppWithBuildDependency.cs

```diff
@@ -17,8 +17,6 @@ public void PublishExcludesBuildDependencies()
             var testInstance = TestAssetsManager.CreateTestInstance("AppWithDirectDependencyAndTypeBuild")
                 .WithLockFiles();
 
-            new RestoreCommand() { WorkingDirectory = testInstance.TestRoot }.Execute().Should().Pass();
```

Comment:
@anurse Any reason why there is this restore command here? We already do a restore for all the TestAssets in our build scripts. This seems redundant and I am planning to remove it.